src/clr-angular/layout/tabs/enums/tabsLayout.ts

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
Filename should be `tabs-layout.enum.ts`

Reply:
omg! of course!